docs/DeveloperGuide.md

```diff
@@ -335,19 +348,51 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. DonnaFin shows an error message.
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**UC03: Locating a contact by name**
```

Comment:
Can change to filter or find. Feels abit weird ngl. But personal preference i guess. Ur call

Reply:
okay changed!